### ISSUE TITLE
Performance tests from Martin Ankerl

### ISSUE THREAD

**Issue by dumblob** [rigtorp/HashMap#2]
How does this hashmap perform with regards to the thorough testing @martinus did in his [blog series](https://martin.ankerl.com/2019/04/01/hashmap-benchmarks-01-overview/ ) about the current cutting edge hash maps?

**Comment by martinus** [comment]
I've tried to add it when creating the benchmark, but unfortunately the map doesn't have a default constructor so I've stopped right there. See https://github.com/martinus/map_benchmark#maps-i-couldnt-add

**Comment by rigtorp** [comment]
Yes I think the empty key should be selected through a template parameter instead of a constructor argument at runtime.
